src/main/java/org/folio/inventory/validation/ItemUpdateValidator.java

```diff
@@ -0,0 +1,45 @@
+package org.folio.inventory.validation;
+
+import java.util.Objects;
+import java.util.Optional;
+
+import org.folio.inventory.domain.items.Item;
+import org.folio.inventory.domain.items.ItemStatusName;
+import org.folio.inventory.support.http.server.ValidationError;
+
+import io.vertx.ext.web.RoutingContext;
+
+public final class ItemUpdateValidator {
```

Comment:
I think we need a better name for this, maybe this could be changed to describe detecting disallowed changes?

Reply:
I've renamed it to `ItemsValidator` and added methods that describes validation rule being asserted.